lib/council-server/src/lib.rs

```diff
@@ -1,12 +1,25 @@
+#![warn(
+    clippy::unwrap_in_result,
+    clippy::unwrap_used,
+    clippy::panic,
+    clippy::panic_in_result_fn
+)]
+
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
+use strum::EnumDiscriminants;
 use ulid::Ulid;
 
 pub mod client;
 pub mod server;
 
+// The subject generator module is for internal use. The module is private as a result.
+mod subject_generator;
+
+pub use client::management::ManagementClient;
 pub use client::{Client, PubClient};
 pub use server::Server;
+pub(crate) use subject_generator::SubjectGenerator;
```

Comment:
Keep the `SubjectGenerator` local to the crate. Should be unused by the outside world.